Update example

jmesnil opened the pull request:
* use the `coolstore` project (instead of `eap` to avoid any confusion
* group together RH-SSO instance and configuration in `sso.yml`
* group together AMQ Broker instance and configuraiton in `am-broker.yaml`
* update `psql.yml` so that it can be properly installed
* remove modules & extensions as the PostgreSQL JDBC driver is now installed using feature pack
* update the ConfigMap with only required env vars
* update helm.yaml to provision EAP using Galleon layers (including PostgreSQL datasource)

deewhyweb commented on openshift/operators/sso-operator.yml:
```diff
@@ -2,20 +2,16 @@ apiVersion: operators.coreos.com/v1
   targetNamespaces:
-  - eap
-
+  - coolstore3
```
This should be coolstore not coolstore3

deewhyweb replied:
Also, I noticed the change from jdk 11 to jdk 8, what is the reason for this?

jmesnil replied:
fixed.

That's annoying, this is the only mention of the namespace that prevents to be able to deploy the app in any namespace. 
Unfortunately, RH-SSO operator does not support installation in all namespace and I've not been able to find a way to make the operator group configuration contextual to its namespace.

deewhyweb reviewed (commented):
Looks good, minor change to sso-operator.yml

jmesnil commented:
> Also, I noticed the change from jdk 11 to jdk 8, what is the reason for this?

That's a mistake from me. When I cleaned up the helm, I remove the `jdk8` fields and thought you wanted to use it. But your Helm configuration clearly specified JDK 11. I've fixed it

deewhyweb commented:
Thanks Jeff, these changes make a huge improvement
